canonical/edgex-device-rfid-llrp-go
# feat(snap): Add snap support for reader discovery

Executing the command edgex-device-rfid-llrp.discover-readers will run a script which
scans for the available subnets and then scans them for available RFID drivers.

This snap requires the network-observe and network-control interfaces to work and
when they have been added a hook runs device discovery automatically.

To test this, do
 snapcraft --use-lxd
 sudo snap install --dangerous *.snap
 sudo snap connect edgex-device-rfid-llrp:network-observe :network-observe
 sudo snap connect edgex-device-rfid-llrp:network-control :network-control

That will run the device discovery and edgex-cli device list should then list
the found devices.

Signed-off-by: Siggi Skulason <siggi.skulason@canonical.com>

## PR Checklist
Please check if your PR fulfills the following requirements:


## PR Type
What kind of change does this PR introduce?




## What is the current behavior?

N/A
Issue Number:


## What is the new behavior?
Device and subnet discovery can be run manually with the command 
edgex-device-rfid-llrp.discover-readers 
or automatically when the required interfaces have been connected

## Does this PR introduce a breaking change?



## Are there any new imports or modules? If so, what are they used for and why?
N/A

## Are there any specific instructions or things that should be known prior to reviewing?
N/A
## Other information
N/A
